lib/SimpleSAML/Modules/Module.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+
+namespace SimpleSAML\Modules;
+
+
+use Symfony\Component\Config\Exception\FileLocatorFileNotFoundException;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Loader\DirectoryLoader;
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+abstract class Module extends Bundle
```

Comment:
This abstract class extends from Symfony Bundle abstract class. So a module is a bundle. Probably we can create a thin abstract class because we don't require all the logic inside the base class.